src/digitalmarketplace/components/header/_header.scss

```
@@ -0,0 +1,29 @@
+@import "../../../govuk-frontend/tools/all";
+@import "../../../govuk-frontend/helpers/all";
+
+@include govuk-exports("digitalmarketplace/component/header") {
+
+
+/* This file extends GOV.UK Frontend header component
+   by changing the blue bottom border to red and shifting
+   the navigation items to the far right.
+*/
+
+  .app-adamin-header {
```

Can this be `admin`?

```suggestion
  .app-admin-header {
```

---

Why? it makes it fun trying to debug a typo 😝

Good spot, I'll also need to update the template and the tests.